starwars/src/components/Characters.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Character from './Character'
+
+const Characters = props => {
+    return (
+        <div>
+            {props.data.map(item => 
+                <Character characterData={item} />
+            )}    
+        </div>
+    );
+};
+
+export default Characters;
```

Comment:
## Assessing your React Learning
 * [x] Answers to your written questions will be recorded in Answers.md
  
## Project
* [x]  Build a card for each object in the starwars array.
MVP achieved!